common/httpx/title.go

```diff
@@ -40,6 +40,25 @@ func ExtractTitle(r *Response) (title string) {
 	return title
 }
 
+func CanHaveTitleTag(mimeType string) bool {
+    mimeTypes := []string{
```

Comment:
Let's move mime types to the beginning of this file as
```go
var (
	cutset                  = "\n\t\v\f\r"
	reTitle                 = regexp.MustCompile(`(?im)<\s*title.*>(.*?)<\s*/\s*title>`)
	reContentType           = regexp.MustCompile(`(?im)\s*charset="(.*?)"|charset=(.*?)"\s*`)
	supportedTitleMimeTypes = []string{
		"text/html",
		"application/xhtml+xml",
		"application/xml",
		"application/rss+xml",
		"application/atom+xml",
		"application/xhtml+xml",
		"application/vnd.wap.xhtml+xml",
	}
)
```
and refactor this function as

```go
func CanHaveTitleTag(mimeType string) bool {
	return slices.Contains(supportedTitleMimeTypes, mimeType)
}
```